src/bootstrap/src/utils/change_tracker.rs

```diff
@@ -173,7 +173,7 @@ pub const CONFIG_CHANGE_HISTORY: &[ChangeInfo] = &[
     ChangeInfo {
         change_id: 123711,
         severity: ChangeSeverity::Warning,
-        summary: "The deprecated field `changelog-seen` has been removed. Using that field in `config.toml` from now on will result in breakage.",
+        summary: "The deprecated field `changelog-seen` has been removed. Using that field in `bootstrap.toml` from now on will result in breakage.",
```

Comment:
We should revert the current changes in `change_tracker.rs` and add a new change entry to inform people about this renaming work.